frontend/src/components/Header/index.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import dots from './assets/dots.png';
+import avatar from './assets/avata1r.png'
+import { Button } from './../shared/button/index.js'
+
+export const HeaderComponent = () => {
+  let avatarImg = {
+    backgroundImage: `url("${avatar}")`
+  }
+  return (
+    <div className="bg-c400 border-b-2 border-c100 " >
+      <header className="w-main-container mx-auto flex flex-row pt-56 ">
+
+        <div className="mb-56 ">
+
+          <div className="  " >
+            <span className=" text-lg font-body-Rubik font-bold text-c100 mb-4 " >I am</span>
+            <h1 className=" text-xxlg font-bold font-headings mb-4 " >Mark Parker</h1>
```

Comment:
heading font-size should be 100px wich equal 6.25rem, Double the size of the font you put in

Reply:
Should I use font sizing in global styling? Or heading font size hasn't global styling